phpunit.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="tests/Bootstrap.php"
+         colors="true"
+         testSuiteLoaderClass="PHPUnit_Runner_StandardTestSuiteLoader"
+         backupGlobals="false"
+         backupStaticAttributes="false">
```

Comment:
You may want to set `forceCoversAnnotation="true"` to force test-methods to be explicit on which code is expected to run.